cfpb/consumerfinance.gov
# Update to latest expandable with flexbox header

## Changes

- Update to latest expandable with flexbox header
- Strip out legacy markup to reflect changes in DS expandable markup from https://github.com/cfpb/design-system/pull/1412


## How to test this PR

1. `yarn && yarn build` and run a local server.
2. Check expandables on the site and see that the toggle doesn't shift when opening/closing, including:
 -  financial well being http://localhost:8000/consumer-tools/financial-well-being/results/
 - iregs, at desktop/mobile http://localhost:8000/rules-policy/regulations/1002/
 - buying a home form explainers https://www.consumerfinance.gov/owning-a-home/loan-estimate/
 - rural or underserved app http://localhost:8000/rural-or-underserved-tool/#rural-or-underserved
 - TDP (not sure where these pages are??)
